src/styles/blocks/card.scss

```diff
@@ -18,6 +18,12 @@
     width: $picture-width;
     height: 134px;
     margin: 9px auto 45px;
+    transition-duration: $transit-time;
+  }
+
+  &__picture:hover {
+    transform: scale(1.2);
+    border: 3px solid #f3f3f3;
```

Comment:
you need to add a transition-property with a value transform because the border should appear immediately

Reply:
add to picture transition with transform value